### PR TITLE
Updates based on feedback

### PR DESCRIPTION
**Pull request by meker12**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

Minor corrections based on review feedback.

## Additional information

*List all affected URLs*

https://devdocs.magento.com/guides/v2.3/cloud/cdn/trouble-fastly.html

### CODE REVIEW

File: guides/v2.1/cloud/cdn/trouble-fastly.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ If the Fastly service returns incorrect headers, submit a Fastly API request dir
 
        ```bash
        curl http[s]://<your domain>.com -H "Host:<URL>" -k -vo /dev/null -H Fastly-Debug:1
```
Comment (hguthrie) [inline guides/v2.1/cloud/cdn/trouble-fastly.md]:
```suggestion
       curl https://<your-domain>.com -H "Host:<URL>" -k -vo /dev/null -H Fastly-Debug:1
```

File: guides/v2.1/cloud/cdn/trouble-fastly.md [hunk 2]
```diff
@@ -129,13 +129,13 @@ Fastly API requests are passed through the Fastly extension to get a response fr
 1.  In a terminal, use the following `curl` command to test your live site URL:
 
     ```bash
-curl http://www.mymagento.biz -vo /dev/null -H Fastly-Debug:1`
+curl https://<live URL> -vo /dev/null -H Fastly-Debug:1
```
Comment (hguthrie) [inline guides/v2.1/cloud/cdn/trouble-fastly.md]:
```suggestion
curl https://<live-URL> -vo /dev/null -H Fastly-Debug:1
```

File: guides/v2.1/cloud/cdn/trouble-fastly.md [hunk 3]
```diff
@@ -129,13 +129,13 @@ Fastly API requests are passed through the Fastly extension to get a response fr
...
     ```
 
 	  If you have not set a static route or completed the DNS configuration for the domains on your live site, use the `--resolve` flag, which bypasses DNS name resolution.
    
     ```bash
-curl http://<live URL> -vo /dev/null -H Fastly-Debug:1 [--resolve]
+curl https://<live URL> -vo /dev/null -H Fastly-Debug:1 [--resolve] <live URL hostname>:443:<live IP address>
```
Comment (hguthrie) [inline guides/v2.1/cloud/cdn/trouble-fastly.md]:
```suggestion
curl https://<live-URL> -vo /dev/null -H Fastly-Debug:1 [--resolve] <live-URL-hostname>:443:<live-IP-address>
```

**Comment by meker12** [comment]
running tests

**Pull request merged**